### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by EpicM147** [Qyfashae/discordid_to_ipaddress#2]
Traceback (most recent call last):
  File "/home/main.py", line 1, in <module>
    import aiohttp
ModuleNotFoundError: No module named 'aiohttp'

**Comment by aazz18** [comment]
```pip install aiohttp```

**Comment by PossiblePanda** [comment]
it's fake, it's not possible to get an IP from ID without insider information

**Comment by aazz18** [comment]
> it's fake, it's not possible to get an IP from ID without insider information

Nahhh we can use the profile ID of a discord account to find IP 💀💀💀 bro imagine if that was real discord would be sued xd

**Comment by PossiblePanda** [comment]
> > it's fake, it's not possible to get an IP from ID without insider information
> 
> Nahhh we can use the profile ID of a discord account to find IP 💀💀💀 bro imagine if that was real discord would be sued xd

IP addresses aren't as important as people say, it's over exaggerated by VPN companies so they can make money. Most routers dynamically reset your ip address after a while

**Comment by aazz18** [comment]
That's true but discord would still be sued by some people who don't really know that or are just stupid like US senators

**Comment by PossiblePanda** [comment]
> That's true but discord would still be sued by some people who don't really know that or are just stupid like US senators

yeah maybe

**Comment by Qyfashae** [comment]
This repo is a little bit of both iconic and ironic repo. 

People think its green text on the screen falling down and taking over social media accounts in one-click's. 
So a little bit just because of that (;

**Comment by PossiblePanda** [comment]
> This repo is a little bit of both iconic and ironic repo.
> 
> People think its green text on the screen falling down and taking over social media accounts in one-click's. So a little bit just because of that (;

yep LMAOO

**Comment by PossiblePanda** [comment]
respect to you for doing that, it's funny

**Comment by jackoil** [comment]
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host discord.com:443 ssl:default [The remote computer refused the network connection]
